pkg/crc/machine/machine.go

```diff
@@ -612,14 +612,19 @@ func GetConsoleURL(consoleConfig ConsoleConfig) (ConsoleResult, error) {
 	// We might need to improve and use crc status logic, only
 	// return if the Openshift is running as part of status.
 	libMachineAPIClient := libmachine.NewClient(constants.MachineBaseDir, constants.MachineCertsDir)
-	_, err := libMachineAPIClient.Load(consoleConfig.Name)
+	host, err := libMachineAPIClient.Load(consoleConfig.Name)
 
 	if err != nil {
 		result.Success = false
 		result.Error = err.Error()
 		return *result, errors.New(err.Error())
 	}
-	result.URL = constants.DefaultWebConsoleURL
+	_, crcBundleMetadata, err := getBundleMetadataFromDriver(host.Driver)
+	if err != nil {
+		result.Error = err.Error()
+		return *result, errors.Newf("Error loading bundle metadata: %v", err)
+	}
+	err = fillClusterConfig(crcBundleMetadata, &result.ClusterConfig)
```

Comment:
What this `err` do if we are not returning it?

Reply:
CI also complained, I've pushed another version of the PR without that problem.